CodeGen/tests/test_codegen_on_gaudi.sh

```diff
@@ -51,7 +53,7 @@ function start_services() {
     # Start Docker Containers
     docker compose -f docker_compose.yaml up -d
 
-    sleep 2m # Waits 2 minutes
+    sleep 5m # Waits 5 minutes
```

Comment:
I don't think you need to expand the sleep time.

Reply:
reverted back to 2m